scripts/BillsHouse.asm

```diff
@@ -95,78 +95,78 @@ MovementData_1e807:
 	db NPC_MOVEMENT_DOWN
 	db -1 ; end
 
-BillsHouseScript4:
+BillsHouseCleanupScript:
 	ld a, [wd730]
 	bit 0, a
 	ret nz
 	xor a
 	ld [wJoyIgnore], a
 	SetEvent EVENT_MET_BILL_2 ; this event seems redundant
 	SetEvent EVENT_MET_BILL
-	ld a, $0
+	ld a, SCRIPT_BILLSHOUSE_DEFAULT
 	ld [wBillsHouseCurScript], a
 	ret
 
-BillsHouseScript5:
-	ld a, $4
+BillsHousePCScript:
+	ld a, TEXT_BILLSHOUSE_ACTIVATE_PC
 	ldh [hSpriteIndexOrTextID], a
```

Comment:
Sounds good. I can do that with a Regex at the end.

Reply:
There are some instances of `hSpriteIndexOrTextID` that ought to be replaced with `hSpriteIndex` though.

Reply:
Intersting... i'll have to look out for those.. and maybe do a manual pass. Do you have a particular example?

Reply:
Well anytime you see `ldh [hSpriteIndexOrTextID], a` followed by `call DisplayTextID` or `call PrintPredefTextID` you know it's a text ID.

When `ldh [hSpriteIndexOrTextID], a` is followed by `call IsSpriteInFrontOfPlayer`, it's a sprite ID. Or when `wMissableObjectList` is involved (like in engine/events/pick_up_item.asm).

Sometimes it's a little messy and not totally clear-cut, like in `IsSpriteOrSignInFrontOfPlayer`.

It could be fuzzy enough that we just put it off until later.

Reply:
Opened an issue at https://github.com/pret/pokered/issues/406. Resolving this.